docs/management/api/_exceptions-api-reusable-content.asciidoc

```diff
@@ -0,0 +1,186 @@
+// --------------------------------------------------------------------------------------------------
+//  ABOUT:
+//  This file contains a set of re-usable content that can be pulled into documentation pages that describe
+//  the API usage for endpoint artifacts (ex. Trusted Apps, event filters, etc...).
+//
+//  In order to use this content, the following must be defined in source document that "include"'s this
+//  content:
+//
+//  ATTRIBUTES:
+//  The following attributes are used through out the content to customize the information to the
+//  specific artifact type:
+//
+//  - :endpoint-artifact-list-id:
+//      The 'list_id' for the artifact type
+//      Example: endpoint_trusted_apps
+//  - attribute - :endpoint-artifact-name:
+//      The name of the artifact type (usually plural)
+//      Example: Trusted Applications
+//  - attribute: :endpoint-artifact-name-single:
+//      The name of the artifact in singular form
+//      Example: Trusted Application
+//  - attribute: :endpoint-artifact-name-sentence-start:
+//      The name of the artifact type (usually plural) with first letter capitalized
+//      Example: Trusted applications
+//  - attribute: :endpoint-artifact-name-single-sentence-start:
+//      The name of the artifact in singular form with first letter capitalized
+//      Example: Trusted application
+//
+//  DOCUMENT IDS:
+//  The reusable content in this file references the following ids which must exist in the document:
+//
+//  - [#create-item-api]
+//  - [#update-item-api]
+//  - [#exception-item-entry-schema]
+//
+// --------------------------------------------------------------------------------------------------
+
+// - - - - - - - - - - - - - - - -
+// A general list of payload properties that must be set when using create/update APIs
+// - - - - - - - - - - - - - - - -
+// tag::endpoint-exceptions-request-payload-requirements[]
+// tag::endpoint-exceptions-request-payload-list-id-and-namespace-requirements[]
```

Comment:
@ashokaditya note that I addressed what you pointed out a little different from what I comment on in your prior feedback. I added this additional tag, inside of the one above, that wraps only the `list_id` and `namesapce_type` bullets. Then, in the Trusted Apps document, I included only this part of it and added the `type` there to say `endpoint`. see below: 

https://github.com/elastic/security-docs/pull/1634/files#diff-9c07987e66cb25711b43c8f9c49d30e237046bbb6e7db577b540bbaf4f3eb8a7R23-R24

Reply:
Nice. DIdn't know we can nest tags! 🔥 I guess this indeed makes you the asciidoc expert now🥇

Reply:
LOL 🤦